src/python/test/sugar/__init__.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import print_function, unicode_literals, division, absolute_import
+import contextlib
+import os
+import shutil
+import subprocess
+import tempfile
```

Comment:
I recommend importing our wrapper tempfile [`dxpy_testutil.TemporaryFile`](https://github.com/dnanexus/dx-toolkit/blob/master/src/python/test/dxpy_testutil.py#L651)

Reply:
That only works for files, not directories. But I agree that these utility functions should at some point be pushed up into dxpy_testutils.py.